front-end/src/components/molecules/side-bar/side-bar.tsx

```diff
@@ -15,6 +19,21 @@ const dashboardElements = [
 ];
 
 export default function Sidebar() {
+  const router = useRouter();
+
+  const handleLogout = () => {
+    localStorage.removeItem("token")
+    router.replace("/");
+    toast.success("Logout successful!", {
+      position: "bottom-right",
+      autoClose: 2000,
+      hideProgressBar: false,
+      closeOnClick: true,
+      pauseOnHover: true,
+      draggable: true,
```

Comment:
you should take reference from the way this toast was implemented in other pages.
Or check it's very easy to navigate documentation for clear guide.
you don't need to add all this.

```suggestion
```